web/src/App.jsx

```
@@ -333,7 +334,19 @@ const DeviceContactItem = memo(({ contact }) => {
   }
 
   return (
-    <div className="contact-row contact-row--stacked">
+    <div
+      className="contact-row contact-row--stacked"
+      onClick={() => onSelect && onSelect(contact)}
+      role="button"
+      tabIndex={0}
+      onKeyDown={(e) => {
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)